packages/terraform-modules/package.json

```diff
@@ -30,13 +30,7 @@
     "parse-domain": "5.0.0"
   },
   "devDependencies": {
-    "@semantic-release/changelog": "6.0.3",
-    "@semantic-release/commit-analyzer": "11.1.0",
-    "@semantic-release/exec": "6.0.3",
-    "@semantic-release/git": "10.0.1",
```

Comment:
Most of these are all included by default in sem release, so no need to also manage their versions here